lib/tbot/service_workload_identity_x509.go

```diff
@@ -296,6 +318,13 @@ func (s *WorkloadIdentityX509Service) render(
 		return trace.Wrap(err, "writing svid trust bundle")
 	}
 
+	crlBytes := crlSet.Marshal()
+	if len(crlBytes) > 0 {
+		if err := s.cfg.Destination.Write(ctx, config.SVIDCRLPemPath, crlBytes); err != nil {
+			return trace.Wrap(err, "writing CRL")
+		}
+	}
```

Comment:
Sort of badly defined legacy behavior, but should `SVIDCRLPemPath` be included in the `Describe()` for the service? I'm... actually not sure what we'd want for optional files like this.